lntest/itest/log_error_whitelist.txt

```diff
@@ -11,8 +11,9 @@
 <time> [ERR] CNCT: ChannelArbitrator(<chan_point>): unable to force close: channel not found
 <time> [ERR] CNCT: ChannelArbitrator(<chan_point>): unable to progress *contractcourt.commitSweepResolver: remote party swept utxo
 <time> [ERR] CNCT: ChannelArbitrator(<chan_point>): unable to progress *contractcourt.htlcOutgoingContestResolver: chain notifier shutting down
+<time> [ERR] CNCT: ChannelArbitrator(<chan_point>): unable to progress *contractcourt.htlcOutgoingContestResolver: htlcswitch shutting down
 <time> [ERR] CNCT: ChannelArbitrator(<chan_point>): unable to progress *contractcourt.htlcOutgoingContestResolver: the client has been shutdown
-<time> [ERR] CNCT: ChannelArbitrator(<chan_point>): unable to progress *contractcourt.htlcOutgoingContestResolver: unable to create pre-image from witness: invalid preimage length of 33, want 32
+<time> [ERR] CNCT: ChannelArbitrator(<chan_point>): unable to progress *contractcourt.htlcOutgoingContestResolver: TxNotifier is exiting
```

Comment:
This error is raised here: https://github.com/Jeiwan/lnd/blob/cnct_breach_retribution_fix/contractcourt/htlc_timeout_resolver.go#L181
As far as I understand, this happens because a node gets terminated before it handles the resolution message.